scripts/reactor_deployment.py

```diff
@@ -152,17 +187,26 @@ def greedy_deployment(df, base_col, ar_dict):
     ar_dict: dictionary
         A dictionary of reactors with information of the form:
         {reactor: [Power (MWe), capacity_factor (%), lifetime (yr)]}
+    dep_start_year: int
+        The year to start the deployment of advanced reactors.
```

Comment:
I can't update the `pandas.DataFrame` part -- but I recommend making the doc string consistently formatted (the below is not the same as earlier versions).